packages/tests/src/e2e/bot/MessageExtensionActionHappyPath.dotnet.tests.ts

```diff
@@ -1,13 +1,16 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * @author Siglud <fanhu@microsoft.com>
+ **/
 import { happyPathTest } from "./BotHappyPathCommon";
 import { Runtime } from "../../commonlib/constants";
 import { it } from "@microsoft/extra-shot-mocha";
 
-describe("Provision message extension Dotnet", () => {
+describe("Provision message extension action Dotnet", () => {
```

Comment:
Do we need to clean up old M365 search test case or old ME test case?

Reply:
No we don't need to. The old test cases still work.